FirebaseAuth/Sources/Storage/FIRAuthKeychainServices.m

```diff
@@ -129,11 +131,18 @@ - (nullable NSData *)itemWithQuery:(NSDictionary *)query error:(NSError **_Nulla
 
   if (status == noErr && result != NULL) {
     NSArray *items = (__bridge_transfer NSArray *)result;
-    if (items.count != 1) {
+    if (items.count == 0) {
       if (error) {
+        // The keychain query returned no error, but there were no items found.
         *error = [FIRAuthErrorUtils keychainErrorWithFunction:@"SecItemCopyMatching" status:status];
       }
       return nil;
+    } else if (items.count > 1) {
+      // More than one keychain item was found, all but the first will be ignored.
+      FIRLogError(
+          kFIRLoggerAuth, @"I-AUT000005",
+          @"Keychain query returned multiple results, all but the first will be ignored: %@",
```

Comment:
Should the match limit of 2 above be removed, in order to get all the ignored results?